include/fix8/message.hpp

```diff
@@ -1141,7 +1141,9 @@ class Message : public MessageBase
 	    \return calculated checknum */
 	static unsigned calc_chksum(const char *from, const size_t sz, const unsigned offset=0, const int len=-1)
 	{
-#if defined FIX8_SIZEOF_UNSIGNED_LONG && FIX8_SIZEOF_UNSIGNED_LONG == 8
+// steroid checksum disabled as its bugged for large messages.
```

Comment:
Can't we just cherry-pick fix for this from master?

Reply:
we could, but not in this build.

this is supposed to mirror what we were using.
and we were not using that commit.

Reply:
ok